Workflow Fail: Extract data from DRTT #2

## Expected Behavior

####  ➡️  Run python `script.py`
```shell
Run python script.py
  python script.py
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/[3](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3855845927/jobs/6571353639#step:5:3).11.1/x6[4](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3855845927/jobs/6571353639#step:5:4)

```

### Actual Behavior

### ➡️ Run python `script.py`
```shell
Run python script.py
  python script.py
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/[3](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3856013359/jobs/6571714706#step:5:3).11.1/x6[4](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3856013359/jobs/6571714706#step:5:4)

Traceback (most recent call last):
  File "/home/runner/work/devo-abastecer/devo-abastecer/script.py", line [5](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3856013359/jobs/6571714706#step:5:5)[8](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3856013359/jobs/6571714706#step:5:8), in <module>
    price = float(gas_info[i + 1][:-1].replace(',', '.'))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: 'Gasolina IO[9](https://github.com/carlosrsabreu/devo-abastecer/actions/runs/3856013359/jobs/6571714706#step:5:9)'
Error: Process completed with exit code 1.
```

## Steps to Reproduce the Problem

>[![Extract data from DRTT](https://github.com/carlosrsabreu/devo-abastecer/actions/workflows/main.yml/badge.svg?branch=main&event=schedule)](https://github.com/carlosrsabreu/devo-abastecer/actions/workflows/main.yml)
1. The issue currently exists in all branches.
2. The issue can be reproduced by running the workflow manually or on schedule.

--- a/script.py
+++ b/script.py
@@ -55,7 +55,14 @@ if update:
     i += 1
     while i < len(gas_info) - 1:
         gas = gas_info[i]
-        price = float(gas_info[i + 1][:-1].replace(',', '.'))
+        price = None
+        # skip to next line in case fuelPrice is not a float
+        while price is None:
+            try:
+                price = float(gas_info[i + 1][:-1].replace(",", "."))
+            except ValueError:
+                i += 1 
+
         dict_prices[CURRENT_WEEK][GAS_KEY][gas] = price
         i += 2
     # Add Gasoline 98 price

